pages/clustering/high-availability.mdx

```diff
@@ -104,13 +104,15 @@ docker run --name coord1 -p 7690:7687 -p 7445:7444 memgraph/memgraph-mage
 --coordinator-id=1 
 --experimental-enabled=high-availability
 --coordinator-hostname=localhost
+--management-port=10121
 ```
 
-Coordinator IDs serve as identifiers, while the coordinator port is used for synchronization and log replication between coordinators. Coordinator IDs and coordinator ports must be different for all coordinators.
+Coordinator IDs serve as identifiers, the coordinator port is used for synchronization and log replication between coordinators and management port is used to get health state of cluster from leader coordinator. 
+Coordinator IDs, coordinator ports and management ports must be different for all coordinators.
 
-Configuration option `--coordinator-hostname` must be set on all coordinator instances. It is used only in the `SHOW INSTANCES` query to correctly return networking information about
-the first leader (the coordinator instance on which you run registration queries). Setting this configuration flag to some random value will not cause issues in the cluster's capabilities
-but you will get misleading output when running `SHOW INSTANCES` query. 
+Configuration option `--coordinator-hostname` must be set on all coordinator instances. It is used on followers to ping the leader coordinator on the correct IP address and return health state about
+the cluster. Setting this configuration flag to some random value will cause issues in the cluster's capabilities
```

Comment:
Before talking about random values, let's give some examples of successful values. Does it need to be an ip address? an FQDN? can it appear in the /etc/hosts file?

Reply:
I changed it, let me know if it is not fine